src/Files.App/Views/Pages/PropertiesGeneral.xaml

```diff
@@ -293,12 +291,13 @@
 				<ColumnDefinition Width="Auto" />
 			</Grid.ColumnDefinitions>
 			<Grid.RowDefinitions>
-				<RowDefinition Height="Auto" />
-				<RowDefinition Height="Auto" />
-				<RowDefinition Height="Auto" />
+				<RowDefinition Height="*" />
+				<RowDefinition Height="*" />
+				<RowDefinition Height="*" />
```

Comment:
Does this need to be changed?

Reply:
This guarantees that the 3 texts use the same height, which was not the case before the correction of the size of the 3rd text. It doesn't change anything in practice anymore because it's fixed, but it says in the code that they should be the same height.

Reply:
What if the text is wrapped onto a different line? I think Auto works better here.

Reply:
You are right. It's back to auto.